src/azul/plugins/metadata/hca/transform.py

```diff
@@ -349,22 +471,31 @@ def _donor_types(cls) -> FieldTypes:
             'biological_sex': null_str,
             'genus_species': null_str,
             'diseases': null_str,
-            'organism_age': null_str,
+            'organism_age': organism_age,
             'organism_age_unit': null_str,
+            'organism_age_value': null_str,
             # Prevent problem due to shadow copies on numeric ranges
             'organism_age_range': pass_thru_json,
             # Pass through field added by DonorOrganismAggregator
             'donor_count': pass_thru_int
         }
 
-    def _donor(self, donor: api.DonorOrganism) -> MutableJSON:
+    def _donor(self, donor: api.DonorOrganism, organism_age: OrganismAge = organism_age) -> MutableJSON:
```

Comment:
The shadowing of `organism_age` is confusing. Why is this parameter needed? What call sites pass it?